### DISCUSSION TITLE
cannot change save location in settings on Mac

### DISCUSSION THREAD

**Discussion by tezb777** [aclap-dev/video-downloadhelper#205] [Help and Q&A]
### Your question

i have changed to a new Mac, all was good on my old Mac but on this Mac i cannot change the location to save the download file to, i want to change the location to a folder i have created on the desktop but when i try to change the destination in settings i get the message in the pic attached, how can i make my downloaded file save to the folder on the desktop ???  thanks in advance
![IMG_20240222_211313](https://github.com/aclap-dev/video-downloadhelper/assets/38738426/bd527145-7a2b-43ed-bb1b-a9a322ee4817)

### Addon & CoApp version and details

![Uploading IMG_20240222_211313.jpg…]()

**Comment by jcfrog** [comment]
Hi,
Sorry it's a rare error we can't reproduce. It's a local configuration issue, maybe user rights problem.

**Comment by paulrouget** [comment]
Oh, first time I see that on MacOS.

I think the CoApp needs special right to read/write on Desktop.

I'll look into it.

**Comment by paulrouget** [comment]
What if you do this:

1. Open Setting > Behavior
2. Click on "Use Native FilePicker"
3. Click "Save"
4. Click on "Change" and choose your directory
5. Click "Save"
